lib/core/src/irods_default_paths.cpp

```diff
@@ -18,31 +21,83 @@ namespace irods {
         try {
             boost::filesystem::path path{dl_info.dli_fname};
             path = boost::filesystem::canonical(path);
-            path.remove_filename().remove_filename().remove_filename(); // Removes filename and the two directories (usr and lib) between libirods_common.so and base of irods install
+            path.remove_filename();
             return path;
         } catch(const boost::filesystem::filesystem_error& e) {
             THROW(-1, e.what());
         }
     }
 
+    boost::filesystem::path
```

Comment:
Given that this is a cpp file, you can replace the full namespace references with aliases. You can declare the alias after the `#includes` like so:
```c++
namespace fs = boost::filesystem;
```
Then, all of the `boost::filesystem` references become `fs`.